src/ts/components/subtitlesettings/fontopacityselectbox.ts

```diff
@@ -1,37 +1,41 @@
-import {SubtitleSettingSelectBoxConfig, SubtitleSettingSelectBox} from './subtitlesettingselectbox';
+import {SubtitleSettingSelectBox} from './subtitlesettingselectbox';
 import {UIInstanceManager} from '../../uimanager';
-import {StorageUtils} from '../../storageutils';
-import {ColorUtils} from '../../colorutils';
 
 /**
  * A select box providing a selection of different font colors.
  */
 export class FontOpacitySelectBox extends SubtitleSettingSelectBox {
 
-  constructor(config: SubtitleSettingSelectBoxConfig) {
-    super(config);
-  }
-
   configure(player: bitmovin.PlayerAPI, uimanager: UIInstanceManager): void {
     super.configure(player, uimanager);
 
-    this.addItem('0.25', '25%');
-    this.addItem('0.50', '50%');
-    this.addItem('0.75', '75%');
-    this.addItem('1', '100%');
+    this.addItem(null, 'default');
+    this.addItem('100', '100%');
+    this.addItem('75', '75%');
+    this.addItem('50', '50%');
+    this.addItem('25', '25%');
 
-    this.selectItem('1');
+    this.onItemSelected.subscribe((sender, key: string) => {
+      this.settingsManager.fontOpacity.value = key;
 
-    if (StorageUtils.hasLocalStorage()) {
-      let color = window.localStorage.getItem('fontColor');
-      if (color != null) {
-        let col = ColorUtils.colorFromCss(color, ColorUtils.foreground);
-        this.selectItem(col.a.toString());
+      // Color and opacity go together, so we need to...
```

Comment:
It *must*, because "default" means basically "whatever is specified as default CSS style". I cannot set only the selected color and keep the default opacity, because the default opacity is specified somewhere in CSS (e.g. a `style` tag in the html document) and I cannot know its value.

Also, a default opacity of 0% does not make sense. When I select a color I want to see it. With a default opacity of 0% I can select any color, do not see anything, and am wondering if this is broken.